frontend/src/pages/SettingsScheduling.tsx

```diff
@@ -536,49 +543,99 @@ const SettingsScheduling = () => {
           </div>
 
           <div className="settings-item">
-            <p>
-              <i>
-                Send notification on completed tasks with the help of the{' '}
-                <a href="https://github.com/caronc/apprise" target="_blank">
-                  Apprise
-                </a>{' '}
-                library.
-              </i>
-            </p>
-
-            <select
-              value={notificationTask}
-              onChange={e => {
-                setNotificationTask(e.currentTarget.value);
-              }}
-            >
-              <option value="">-- select task --</option>
-              <option value="update_subscribed">Rescan your Subscriptions</option>
-              <option value="extract_download">Add to download queue</option>
-              <option value="download_pending">Downloading</option>
-              <option value="check_reindex">Reindex Documents</option>
-            </select>
-
-            <input
-              type="text"
-              placeholder="Apprise notification URL"
-              value={notificationUrl}
-              onChange={e => {
-                setNotificationUrl(e.currentTarget.value);
-              }}
-            />
-
-            <Button
-              label="Save"
-              onClick={async () => {
-                await createAppriseNotificationUrl(
-                  notificationTask as AppriseTaskNameType,
-                  notificationUrl || '',
-                );
+            <div>
+              <p>
+                <i>
+                  Send notification on completed tasks with the help of the{' '}
+                  <a href="https://github.com/caronc/apprise" target="_blank">
+                    Apprise
+                  </a>{' '}
+                  library.
+                </i>
+              </p>
+
+              <select
+                value={notificationTask}
+                onChange={e => {
+                  setNotificationTask(e.currentTarget.value);
+                }}
+              >
+                <option value="">-- select task --</option>
+                <option value="update_subscribed">Rescan your Subscriptions</option>
+                <option value="extract_download">Add to download queue</option>
+                <option value="download_pending">Downloading</option>
+                <option value="check_reindex">Reindex Documents</option>
+              </select>
+
+              <input
```

Comment:
I'd love to get this input wider if anyone has any suggestions on how to get it to fill without pushing other elements onto the next line.